README.md

```diff
@@ -57,58 +55,46 @@
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://redux.js.org/">Redux.js</a></li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+- **[Clicking the 'load/refresh' button gives the user 10 random pictures]**
+- **[Clicking the 'details' button navigates the user to a new page with more information]**
+- **[Clicking the 'back' button on the details page navigates the user back to the previously displayed 10 pictures]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Project Documentation <a name="documentation"></a>
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- [Loom video presenting the project](https://www.loom.com/share/f41f5a44d04847b1a62547194ff417b2)
 
```

Comment:
- Hey, great job with adding the video, however, according to the video requirement, the maximum length of the video should be 5 minutes and your video's length is 6mins. Please, kindly record another one that meets this part of the project requirement. You can check [project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) to gain more clarity.

Reply:
### TESTS

- Great job with the test, however, your tests are failing, kindly please fix this issue :pray:

Reply:
@zieeco can you please show me, how the tests are failing? they are perfect for me.

![image](https://user-images.githubusercontent.com/103699974/204940010-2cb0df39-f340-4963-93bc-5d1aa1f3186b.png)

Reply:
Hey @MattGomb , See it below


![Screenshot from 2022-12-01 02-00-08](https://user-images.githubusercontent.com/59397159/204940496-8ec53442-dca3-4529-898a-9a01a32af643.png)

Reply:
Reference routes are fixed in tests (it was a capital letter issue)

Reply:
New video under 5 minutes is done! :)